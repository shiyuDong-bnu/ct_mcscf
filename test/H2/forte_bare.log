2025-02-18:16:45:40,939 INFO     [psi4.driver.task_planner:285] PLANNING Atomic:  keywords={'D_CONVERGENCE': 1e-08, 'E_CONVERGENCE': 1e-10, 'MAXITER': 300, 'REFERENCE': 'RHF', 'SCF_TYPE': 'PK', 'forte__job_type': 'MCSCF_TWO_STEP', 'forte__scf_type': 'PK', 'forte__e_convergence': 1e-08, 'forte__d_convergence': 1e-08, 'forte__active_space_solver': 'FCI', 'forte__active': [4], 'forte__int_type': 'FCIDUMP', 'forte__fcidump_file': 'BARE.DUMP', 'forte__r_convergence': 1e-08, 'forte__casscf_micro_maxiter': 4, 'forte__casscf_g_convergence': 1e-06, 'forte__reference': 'RHF', 'forte__maxiter': 300, 'forte__basis': 'CC-PVDZ', 'function_kwargs': {}}
2025-02-18:16:45:40,941 INFO     [psi4.driver.driver:523] Compute energy(): method=forte, basis=cc-pvdz, molecule=CO, nre=0.52917721067
2025-02-18:16:45:41,51 INFO     [psi4.driver.driver:526] Return energy(): -1.1327253348615012
