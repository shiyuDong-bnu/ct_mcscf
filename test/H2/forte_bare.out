
    -----------------------------------------------------------------------
          Psi4: An Open-Source Ab Initio Electronic Structure Package
                               Psi4 1.9.1 release

                         Git: Rev {} zzzzzzz 


    D. G. A. Smith, L. A. Burns, A. C. Simmonett, R. M. Parrish,
    M. C. Schieber, R. Galvelis, P. Kraus, H. Kruse, R. Di Remigio,
    A. Alenaizan, A. M. James, S. Lehtola, J. P. Misiewicz, M. Scheurer,
    R. A. Shaw, J. B. Schriber, Y. Xie, Z. L. Glick, D. A. Sirianni,
    J. S. O'Brien, J. M. Waldrop, A. Kumar, E. G. Hohenstein,
    B. P. Pritchard, B. R. Brooks, H. F. Schaefer III, A. Yu. Sokolov,
    K. Patkowski, A. E. DePrince III, U. Bozkaya, R. A. King,
    F. A. Evangelista, J. M. Turney, T. D. Crawford, C. D. Sherrill,
    J. Chem. Phys. 152(18) 184108 (2020). https://doi.org/10.1063/5.0006002

                            Additional Code Authors
    E. T. Seidl, C. L. Janssen, E. F. Valeev, M. L. Leininger,
    J. F. Gonthier, R. M. Richard, H. R. McAlexander, M. Saitow, X. Wang,
    P. Verma, M. H. Lechner, A. Jiang, S. Behnle, A. G. Heide,
    M. F. Herbst, and D. L. Poole

             Previous Authors, Complete List of Code Contributors,
                       and Citations for Specific Modules
    https://github.com/psi4/psi4/blob/master/codemeta.json
    https://github.com/psi4/psi4/graphs/contributors
    http://psicode.org/psi4manual/master/introduction.html#citing-psifour

    -----------------------------------------------------------------------


    Psi4 started on: Tuesday, 18 February 2025 04:45PM

    Process ID: 2703971
    Host:       debianlab
    PSIDATADIR: /home/sydong/miniconda3/envs/p4env/share/psi4
    Memory:     500.0 MiB
    Threads:    1
    
  ==> Input File <==

--------------------------------------------------------------------------
import forte

molecule CO{
  0 1
  H 
  H  1 1.0
  symmetry c1
}

set {
  basis                 cc-pvdz
  reference             rhf
  scf_type              pk
  maxiter               300
  e_convergence         10
  d_convergence         8
#  docc                  [5,0,1,1]
}

set forte {
  job_type              mcscf_two_step
  active_space_solver                fci
#  restricted_docc       [1]
  active                [4,]
  int_type              fcidump
  fcidump_file          BARE.DUMP 
  e_convergence         8  # energy convergence of the FCI iterations
  r_convergence         8  # residual convergence of the FCI iterations
  casscf_e_convergence  8  # energy convergence of the MCSCF iterations
  casscf_g_convergence  6  # gradient convergence of the MCSCF iterations
  casscf_micro_maxiter  4  # do at least 4 micro iterations per macro iteration
}

Eforte = energy('forte')
--------------------------------------------------------------------------

Scratch directory: /tmp/

  Forte
  ----------------------------------------------------------------------------
  A suite of quantum chemistry methods for strongly correlated electrons

    git branch:  - git commit: 

  Developed by:
    Francesco A. Evangelista, Chenyang Li, Kevin P. Hannon,
    Jeffrey B. Schriber, Tianyuan Zhang, Chenxi Cai,
    Nan He, Nicholas Stair, Shuhe Wang, Renke Huang
  ----------------------------------------------------------------------------

  Forte will use custom integrals
  Preparing forte objects from a custom source

  Reading integral information from FCIDUMP file BARE.DUMP
  Read options for space ACTIVE

  ==> MO Space Information <==

  -------------------------------
                        A   Sum
  -------------------------------
    FROZEN_DOCC         0     0
    RESTRICTED_DOCC     0     0
    GAS1                4     4
    GAS2                0     0
    GAS3                0     0
    GAS4                0     0
    GAS5                0     0
    GAS6                0     0
    RESTRICTED_UOCC     6     6
    FROZEN_UOCC         0     0
    Total              10    10
  -------------------------------

  ==> Integral Transformation <==

  Number of molecular orbitals:                         10
  Number of correlated molecular orbitals:              10
  Number of frozen occupied orbitals:                    0
  Number of frozen unoccupied orbitals:                  0
  Two-electron integral type:                       Custom


  Timing for freezing core and virtual orbitals:              0.000 s.
  Timing for preparing custom (FCIDUMP) integrals:            0.000 s.

          -----------------------------------------------------------
                  Multi-Configurational Self Consistent Field
                Two-Step Approximate Second-Order AO Algorithm
            written by Chenyang Li, Kevin P. Hannon, and Shuhe Wang
          -----------------------------------------------------------


  ==> MCSCF Calculation Information <==

    --------------------------------------------------------
    Print level                                      Default
    Integral type                                    FCIDUMP
    CI solver type                                       FCI
    Final orbital type                             CANONICAL
    Derivative type                                     NONE
    Optimize orbitals                                   TRUE
    Include internal rotations                         FALSE
    Debug printing                                     FALSE
    Energy convergence                             1.000e-08
    Gradient convergence                           1.000e-06
    Max value for rotation                         2.000e-01
    Max number of macro iterations                       100
    Max number of micro iterations                         4
    Min number of micro iterations                         4
    DIIS start                                            15
    Min DIIS vectors                                       3
    Max DIIS vectors                                       8
    Frequency of DIIS extrapolation                        1
    --------------------------------------------------------

  ==> Independent Orbital Rotations <==

    ORBITAL SPACES                         A
    ----------------------------------------
             ACTIVE / RESTRICTED_DOCC      0
    RESTRICTED_UOCC /          ACTIVE     24
    RESTRICTED_UOCC / RESTRICTED_DOCC      0
    ----------------------------------------

  ==> MCSCF Iterations <==

                      Energy CI                    Energy Orbital
           ------------------------------  ------------------------------
    Iter.        Total Energy       Delta        Total Energy       Delta  Orb. Grad.  Micro
    ----------------------------------------------------------------------------------------
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
       2      -1.129544952365 -1.1295e+00     -1.132468266161 -1.1325e+00  1.5211e-02    4/N
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
       3      -1.132537840692 -2.9929e-03     -1.132554115863 -8.5850e-05  5.6433e-04    4/N
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
       4      -1.132555612024 -1.7771e-05     -1.132572528355 -1.8412e-05  5.6589e-04    4/N
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
       5      -1.132574451933 -1.8840e-05     -1.132594984924 -2.2457e-05  9.9306e-05    4/N
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
       6      -1.132597155180 -2.2703e-05     -1.132619723481 -2.4739e-05  2.3282e-04    4/N
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
       7      -1.132622545608 -2.5390e-05     -1.132642465250 -2.2742e-05  3.1445e-04    4/N
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
       8      -1.132644497212 -2.1952e-05     -1.132661802072 -1.9337e-05  3.2849e-04    4/N
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
       9      -1.132663700316 -1.9203e-05     -1.132677218212 -1.5416e-05  2.9534e-04    4/N
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
      10      -1.132678559458 -1.4859e-05     -1.132688912327 -1.1694e-05  2.6946e-04    4/N
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
      11      -1.132690011063 -1.1452e-05     -1.132697146305 -8.2340e-06  2.4631e-04    4/N
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
      12      -1.132697861489 -7.8504e-06     -1.132706478091 -9.3318e-06  2.1546e-04    4/N
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
      13      -1.132707263720 -9.4022e-06     -1.132711874861 -5.3968e-06  1.6589e-04    4/N
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
      14      -1.132712291852 -5.0281e-06     -1.132714369453 -2.4946e-06  4.1967e-05    4/N
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
      15      -1.132714403119 -2.1113e-06     -1.132716067401 -1.6979e-06  3.6601e-05    4/N
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
      16      -1.132716103893 -1.7008e-06     -1.132717459684 -1.3923e-06  6.3276e-06    4/N   S
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
      17      -1.132717499622 -1.3957e-06     -1.132718635564 -1.1759e-06  2.8570e-05    4/N   S
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
      18      -1.132718681440 -1.1818e-06     -1.132719634020 -9.9846e-07  5.4079e-05    4/N   S
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
      19      -1.132719684147 -1.0027e-06     -1.132720513666 -8.7965e-07  7.1639e-05    4/N   S/E
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
      20      -1.132724209933 -4.5258e-06     -1.132724737991 -4.2243e-06  5.4028e-05    4/N   S/E
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
      21      -1.132725246771 -1.0368e-06     -1.132725325870 -5.8788e-07  7.5329e-06    4/N   S/E
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
      22      -1.132725317446 -7.0675e-08     -1.132725331270 -5.3994e-09  4.2400e-06    4/N   S/E
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
      23      -1.132725334628 -1.7182e-08     -1.132725334797 -3.5274e-09  9.8076e-07    3/Y   S/E
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
  Integrals are about to be updated.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.001 s.
      24      -1.132725334855 -2.2671e-10     -1.132725334861 -6.3571e-11  6.4384e-08    2/Y
    ----------------------------------------------------------------------------------------

  A miracle has come to pass: MCSCF iterations have converged!

  Performing final CI Calculation using converged orbitals

  ==> String Lists <==

    --------------------------------------------------------
    number of alpha electrons                              1
    number of beta electrons                               1
    number of alpha strings                                4
    number of beta strings                                 4
    --------------------------------------------------------

  ==> FCI Solver <==

    --------------------------------------------------------
    Spin adapt                                         FALSE
    Number of determinants                                16
    Symmetry                                               0
    Multiplicity                                           1
    Number of roots                                        1
    Target root                                            0
    --------------------------------------------------------

  ==> Initial Guess <==

  Initial guess determinants:         16

  Classification of the initial guess solutions

  Number   2S+1   Selected
  ------------------------
    10       1       *
     6       3        
  ------------------------

    Spin    Root           Energy        <S^2>    Status
  -------------------------------------------------------
   singlet    0       -1.132725334862  +0.000000  added
  -------------------------------------------------------

  ==> Root No. 0 <==

    2000      0.98589000
    0200     -0.11088959
    0b0a     -0.06866022
    0a0b     -0.06866022
    0002     -0.05673394
    0020     -0.05374672

    Total Energy:      -1.132725334862, <S^2>: 0.000000
    Time for FCI:       0.000135760000

  ==> Energy Summary <==

    Multi.(2ms)  Irrep.  No.               Energy      <S^2>
    --------------------------------------------------------
       1  (  0)     A     0       -1.132725334862   0.000000
    --------------------------------------------------------

  Time to prepare integrals:        0.002 seconds
  Time to run job          :        0.106 seconds
  Total                    :        0.109 seconds

    Psi4 stopped on: Tuesday, 18 February 2025 04:45PM
    Psi4 wall time for execution: 0:00:00.16

*** Psi4 exiting successfully. Buy a developer a beer!
